addons/hide-delete-button/addon.json

```diff
@@ -2,6 +2,12 @@
   "name": "Hide delete button",
   "description": "Hides delete button (trash can icon) from sprites, costumes and sounds. They can still be deleted using the right click context menu.",
   "tags": ["editor", "featured"],
+  "info": [
+    {
+      "text": "Tip: If you accidentally delete a sprite, costume, or sound, you can undo the deletion by clicking Edit in the menu bar then Restore <item>.",
```

Comment:
Angled brackets can confuse Transifex: #4082

Reply:
Thanks a lot for the reminder!

Reply:
Is there any other way to encode those characters without them messing anything up?

Otherwise, there are some alternatives to this: changing the bracket style, capitalizing "ITEM", or not including the variable "item" in this sentence at all.

Reply:
I'd say we omit the word "item"